### ISSUE TITLE
Youtube embedded videos size

### ISSUE THREAD

**Issue by reallyuniquename** [YePpHa/YouTubeCenter#1971]
Is there any option to force embedded videos size (e.g. large (1024x576) medium (720x405)

**Comment by ireun** [comment]
What do you mean by the player size? I don't think it's possible, since it's defined in the html if the page, and YTC don't have access to this.

**Comment by reallyuniquename** [comment]
Basicly it's size of the iframe that contains embedded YouTube player.
>it's defined in the html if the page, and YTC don't have access to this.

Can't you change iframe's size from the inside?

I know few user javascripts that change embedded player width/height but I don't really want to use them because it's only one part of their functionality and rest ones overlap with YTCenter.

**Comment by ireun** [comment]
@Yonezpt ?

**Comment by Yonezpt** [comment]
It is not up to me, but from inside a frame I doubt its size can be modified unless it is from the sabe domain. Since embbeded videos are by nature iframes from YouTube inside other domains, the CORS policy should deny access to the IFRAME resizing methods. The only way of doing this would have to be by setting YTC to run on all pages and check each page individually for iframe videos and then modify them accordingly. I suspect this will have a serious negative impact on the overall browser experience, not a feature worth implementing.

A much better alternative would be to use stylish to change the iframes.

**Comment by reallyuniquename** [comment]
Oh well. Can you share any stylish or other solution that will force embedded player size?

**Comment by Yonezpt** [comment]
@reallyuniquename That is up to you because I have no idea which sizes you want to change it to nor which pages you want the sizes changed, or if it is for all of them. Either way this is not an appropriate place to ask for this kind of help since this is an issue forum for YTC only.

**Comment by reallyuniquename** [comment]
Let me get this straight. There is no way this feature will be in YTC, right?

**Comment by Yonezpt** [comment]
No, that is up to the script author, he might be interested in implementing it, so you'll have to wait for his input.

**Comment by ireun** [comment]
@Yonezpt I know that it's not up to you, you just understand this batter than me, that's why sometimes i'm mentioning you. :)

**Comment by reallyuniquename** [comment]
Are you guys sure user scripts can't change iframe size from the inside? I mean Greasemonkey has unlimited permissions on changing pages and probably can override cross-origin policy and stuff.

**Comment by Yonezpt** [comment]
Do not mistake XHR requests with CORS permissions. Yes, for requests Greasemonkey provides the ability to overcome the different domain barrier, but not between two iframes. The only way of doing this is to have the script runing on the pages that have the iframes, which means having the script running for all pages until the iframes are found and then change them accordingly.
